### ISSUE TITLE
Non-Home Office police forces

### ISSUE THREAD

**Issue by andrewdwilliams** [openpolitics/manifesto#373]
Something that features very little in public debate and political policy making is the existence of non-Home Office police forces. What I mean by that, is, essentially, all the police forces that are not allocated to a large geographical area like the Metropolitan Police, North Yorkshire Police, Police Scotland and whatever else there is out there.

## Non-Home Office police forces

A number of these non-Home Office police forces are quite notable:

* Ministry of Defence Police
* British Transport Police
* Civil Nuclear Constabulary
* Royal Military Police
* Royal Navy Police
* Royal Air Force Police

Most of them seem quite reasonable and they all have a purpose. But if you investigate further, you find a number of other non-Home Office police forces, the purposes of which seem quite inane. These include the ports police, such as the Port of Bristol Police and the Port of Dover Police and the parks police, such as Kew Constabulary and the Parks Police Service. And there's also a few more, including the Belfast International Airport Constabulary, the Cambridge University Constabulary, the Mersey Tunnels Police and the Northern Ireland Security Guard Service.

## The point

What we have to ask, for a start, is whether it would be more practical to merge, for example, the port police services into their local territorial police force, such as merging the Port of Dover Police into the Kent Police. Surely this would cut costs significantly, especially if the expertise, equipment and facilities of the PoDP transferred with them?

Secondly, I think it might be worthwhile spending some time examining the MoDP, BTP and CNC to look at if they are operating how they should be, whether they are value for money, and whether there have been frequent allegations of wrongdoing against them, possibly underlining the fact they are rarely in action. Essentially, I think it is a good idea to apply a critical mind to these police forces to examine their purpose and whether we ought to keep them.

Oh, and we should definitely disband the Mersey Tunnels Police. I'd like some other peoples input on this.

**Comment by philipjohn** [comment]
Wow, I had no idea these existed!

It'd definitely be good to learn more about them.

**Comment by andrewdwilliams** [comment]
Yeah, I'll definitely look into them a bit further when I get the chance.
